src/renderer/i18n/en/sidebar.ts

```
@@ -0,0 +1,13 @@
+
+const sidebar: SidebarMessages = {
+  'sidebar.tooltip.website': 'Asgardex Website',
+  'sidebar.tooltip.github': 'Asgardex GitHub',
+  'sidebar.tooltip.discord': 'Asgardex Discord',
+  'sidebar.tooltip.twitter': 'Asgardex X',
+  'sidebar.tooltip.license': 'MIT License',
+  'sidebar.tooltip.commitHash': 'Commit Hash',
+  'sidebar.tooltip.playground': 'Playground'
+}
+
+export default sidebar
```

ESLint

[error] Insert `⏎`

(prettier/prettier)